synfig-studio/src/gui/dials/toggleducksdial.cpp

```diff
@@ -55,12 +55,12 @@ using namespace studio;
 
 ToggleDucksDial::ToggleDucksDial(const Gtk::IconSize &size)
 {
-	init_label_button(ducks_position, size, "synfig-toggle_duck_position", _("Position handles"), _("Toggle position handles"));
-	init_label_button(ducks_vertex,   size, "synfig-toggle_duck_vertex",   _("Vertex handles"),   _("Toggle vertex handles"));
-	init_label_button(ducks_tangent,  size, "synfig-toggle_duck_tangent",  _("Tangent handles"),  _("Toggle tangent handles"));
-	init_label_button(ducks_radius,   size, "synfig-toggle_duck_radius",   _("Radius handles"),   _("Toggle radius handles"));
-	init_label_button(ducks_width,    size, "synfig-toggle_duck_width",    _("Width handles"),    _("Toggle width handles"));
-	init_label_button(ducks_angle,    size, "synfig-toggle_duck_angle",    _("Angle handles"),    _("Toggle angle handles"));
+	init_label_button(ducks_position, size, "synfig-toggle_duck_position", _("Position handles"), _("Toggle Position Handles"));
+	init_label_button(ducks_vertex,   size, "synfig-toggle_duck_vertex",   _("Vertex handles"),   _("Toggle Vertex Handles"));
```

Comment:
Why did you uppercase these tooltips but not previous ones? Like those in (new) line 1453 in canvasview.cpp:
```c++
	action_group->add( Gtk::Action::create("save", Gtk::StockID("synfig-save"), _("Save"), _("Save document")),
		hide_return(sigc::mem_fun(*get_instance().get(), &Instance::save))
	);
	action_group->add( Gtk::Action::create("save-as", Gtk::StockID("synfig-save_as"), _("Save As..."), _("Save document As")),
```

Reply:
This change is up to the style we want to use.
In this case I followed the style used in the menus.

Once the style we want to use is determined I can go through all the menu entries and tooltips to make them consistent 👍 

Let's say we have something like:
- "Toggle snap to Grid" it becomes "Toggle Snap to Grid"
- "Window > Workspace > Save workspace..." it becomes "Window > Workspace > Save Workspace..."

Here I've set all in uppercase except adverbs/conjunctions.
I'm not a fan of all uppercase text, but I've noticed that quite common in English UIs.

Reply:
Oh, that Menu Item text then. Sorry :) I thought it was a tooltip. ;)

Reply:
It's a tooltip IIRC but I copied the string used in the menu, anyway I'll check and make sure it's consistent.